backend/app/services/scheduler.py

```
@@ -0,0 +1,233 @@
+import asyncio
+import schedule
+import time
+from datetime import datetime, timedelta
+from typing import List
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)